docs/language/operators/summarize.md

```diff
@@ -5,26 +5,29 @@
 ### Synopsis
 
 ```
-summarize [<field>:=]<agg> [where <expr>][, [<field>:=]<agg> [where <expr>]] [by [<field>][:=<expr>] ...]
+summarize [<field>:=]<agg> [where <expr>][, [<field>:=]<agg> [where <expr>] ...] [by [<field>][:=<expr>] ...]
 ```
 ### Description
 
-The `summarize` operator consumes all of its input, applies an aggregate function
+The `summarize` operator consumes all of its input, applies an [aggregate function](../aggregates/README.md)
 to each input value optionally organized with the group-by keys specified after
-the `by` keyword and at the end of input, and produces one or more aggregations
+the `by` keyword, and at the end of input produces one or more aggregations
 for each unique set of group-by key values.
 
-Each aggregate function may be optionally followed by a `where` clause, which
-indicates a Boolean expression that indicates, for each input value,
-whether to deliver it to that aggregate.
+The keyword `summarize` can be treated as an [implied operator](../overview.md#26-implied-operators)
+and hence may be omitted before an aggregate function call.
```

Comment:
Nit: Copy the corresponding language that appears in put.md.
```suggestion
The `summarize` keyword is optional since it is an
[implied operator](../overview.md#26-implied-operators).
```